apluslms_roman/utils/path_mapping.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import PurePosixPath
+import json
+
+
+def get_host_path(original, mapping):
+    ret = original
+    orig_path = PurePosixPath(original)
+    for k, v in mapping.items():
+        try:
+            print(k, v)
+            relative_path = orig_path.relative_to(k)
+            ret = PurePosixPath(v).joinpath(relative_path)
+            return str(ret)
+        except ValueError:
+            print("Error when composing new path!")
+            return -1
+    return str(ret)
+
+
+def get_pair_form_env(key, json_str):
+    if key == "DOCKER_DIRECTORY_MAP":
```

Comment:
Utility functions should be generic, like functions in standard library. They take input and create output and don't have special cases. This function does something only, when it's argument has specific value. E.g. this if statement should be where this function is called.

Basically, utility function should be testable with it's own unit tests.

Reply:
I'll change to 
```def get_pair_form_env(key, json_str, read_key="DOCKER_DIRECTORY_MAP"):
    if key == read_key:```

Reply:
FYI: The default value for a keyword argument should nearly always be `None`. For int values zero might be ok. Using Python functions that have more complex defaults becomes easily complex.

for reference: https://graysonkoonce.com/always-use-none-for-default-args-in-python/